src/main/java/ca/mcgill/mcb/pcingola/snpEffect/commandLine/SnpEffCmdEff.java

```diff
@@ -629,17 +644,29 @@ else if (outFor.equals("GATK")) {
 						} else if (outFor.equals("TXT")) usage("Output format 'TXT' has been deprecated. Please use 'VCF' instead.\nYou can extract VCF fields to a TXT file using 'SnpSift extractFields' (http://snpeff.sourceforge.net/SnpSift.html#Extract).");
```

Comment:
-csvStats and -htmlStats are optional.  If they are not on the command line, then no stats are output.